scripts/quickstart.py

```diff
@@ -24,9 +26,20 @@
     storage_options={"anon": True},
 ).persist()
 
+# perform groupby aggregation
 result = ddf.groupby("passenger_count").tip_amount.mean().compute()
-
 print(result)
 
+# write result to S3
+result = dd.from_pandas(pd.DataFrame(result), npartitions=1)
```

Comment:
we don't need to do this step. we can do it directly from the result in the line above and just avoid the `.compute()` . I know lots of people do this, but this is not the recommended thing to do.

We only did the compute because it was something simple to show, but if the plan is writing this to_parquet , we would do it directly from the dask dd

Reply:
Yes, fair point. Thanks for spotting this one!